pkg/storaged/block/format-dialog.jsx

```diff
@@ -450,6 +458,12 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                     partition_type = "21686148-6449-6e6f-744e-656564454649";
                 }
 
+                if (type == "swap") {
+                    partition_type = (block_ptable && block_ptable.Type == "dos"
+                        ? "0x82"
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240118-085557-ffd6b834-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)